### ISSUE TITLE
Questions on sensitivity of results to package versions

### ISSUE THREAD

**Issue by sungsoo-ahn** [snap-stanford/pretrain-gnns#35]
Hi, 

Thank you for the great work! I am currently trying to reproduce your results. Everything is going great, except for one dataset (Clintox). Would be able to guide me on this? 

In particular, I am currently finetuning the models that you provided e.g., pretrained_models/gin_contextpred.pth. I am using a finetuning code almost identical to yours (changes only made for logging and refactoring purpose). 

This is the finetuning results that I get for finetuning gin_mask.pth and gin_contextpred.pth, averaged over 5 runs:

 datasets     | bbbp  | clintox | sider | tox21 | bace | toxcast
masking      | 0.648 | 0.713 | 0.602 | 0.765 | 0.786 | 0.633
contextpred | 0.687 | 0.600 | 0.602 | 0.750 | 0.797 | 0.636

The numbers reported in your paper are as follows:

datasets | bbbp | clintox | sider | tox21 | bace | toxcast
masking      | 0.643 | 0.718 | 0.610 | 0.767 | 0.793 | 0.642
contextpred | 0.680 | 0.659 | 0.609 | 0.757 | 0.796 | 0.639

As you can see, the contextpred algorithm for Clintox dataset degrades significantly (0.659 -> 0.609). The 0.609 result has with very small standard deviation (it is an average of 0.614, 0.619, 0.609, 0.567, 0.593). 

I suspect that the main cause if from different package versions that I use. I am curious if you have any knowledge of how the  performance of algorithms are sensitive to the package versions. 

I have attached my finetuning code and Dockerfile for your information. I created the Dockerfile to match library of a recent paper (https://openreview.net/forum?id=DAaaaqPv9-q), but it differs slightly from yours.

[files.zip](https://github.com/snap-stanford/pretrain-gnns/files/6390582/files.zip)

Thank you for your time! I really appreciate it.

**Comment by weihua916** [comment]
Hi! I am not sure if it is because of the package version given that you were able to reproduce the other results well (I have edited your bbbp results from the paper). I am not really sure to be honest.

**Comment by sungsoo-ahn** [comment]
Thanks very much! I was just curious if you have seen any case where the package version would impact the result of the paper significantly.

I think this would be more of a random seed problem rather than a package issue.
